app/views/notifications/index.html.erb

```diff
@@ -16,22 +16,22 @@
         <%= render partial:"notifications_links", locals:{notification:notification} %></li>
       </div>
 
-    <%elsif notification.target_type==="follow"%>
+    <%elsif notification.target_type===Notification.NOTIFICATION_TYPES[:FOLLOW]%>
     <div class="notification">
       <span><strong>Follow's Notification</strong></span>
       <li><%= render partial:'link_to_notification', locals:{notification:notification,message:" sent a follow request",path:current_user}%>
       <%= render partial:"notifications_links", locals:{notification:notification} %></li>
     </div>
 
```

Comment:
This code is repeating itself. Make a partial and pass this changeable data as variable.

Reply:
Done